profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info

```diff
@@ -0,0 +1,260 @@
+features[variable][] = comment_newsroom_selection
+features[variable][] = comment_preview_newsroom_selection
+features[variable][] = comment_subject_field_newsroom_selection
+features[variable][] = date_format_date_only
+features[variable][] = date_format_minimal
+features[variable][] = date_format_short_newsroom
+features[variable][] = field_bundle_settings_node__newsroom_item
+features[variable][] = field_bundle_settings_node__newsroom_selection
+features[variable][] = i18n_sync_node_type_newsroom_selection
+features[variable][] = language_content_type_newsroom_item
+features[variable][] = language_content_type_newsroom_selection
+features[variable][] = menu_options_newsroom_item
+features[variable][] = menu_options_newsroom_selection
+features[variable][] = menu_parent_newsroom_item
+features[variable][] = menu_parent_newsroom_selection
+features[variable][] = node_options_newsroom_item
+features[variable][] = node_options_newsroom_selection
+features[variable][] = node_preview_newsroom_item
+features[variable][] = node_preview_newsroom_selection
+features[variable][] = node_preview_page
+features[variable][] = node_submitted_newsroom_item
+features[variable][] = node_submitted_newsroom_selection
+features[variable][] = pathauto_node_newsroom_item_pattern
+features[views_view][] = newsletter_subscription
+features[views_view][] = newsroom
+features[views_view][] = newsroom_calendar_navigation
+features[views_view][] = newsroom_navigation
+features[views_view][] = newsroom_page_content
+features[views_view][] = newsroom_services
+features[views_view][] = newsroom_term_selection
+features[views_view][] = newsroom_type_featured_item
+features_exclude[dependencies][field_group] = field_group
+features_exclude[dependencies][cce_basic_config] = cce_basic_config
+features_exclude[dependencies][link] = link
+features_exclude[dependencies][media] = media
+features_exclude[dependencies][image] = image
+features_exclude[dependencies][list] = list
+features_exclude[dependencies][file] = file
+features_exclude[dependencies][email] = email
+features_exclude[dependencies][fullcalendar] = fullcalendar
+features_exclude[dependencies][entityreference] = entityreference
+features_exclude[dependencies][date] = date
+features_exclude[dependencies][nexteuropa_multilingual] = nexteuropa_multilingual
+features_exclude[dependencies][date_views] = date_views
+features_exclude[dependencies][features] = features
+features_exclude[dependencies][context] = context
+features_exclude[dependencies][file_entity] = file_entity
+features_exclude[dependencies][strongarm] = strongarm
+features_exclude[field_base][field_tags] = field_tags
+features_exclude[field_base][body] = body
+features_exclude[field_base][name_field] = name_field
+files[] = tests/nexteuropa_newsroom.test
+files[] = views/nexteuropa_newsroom_handler_argument_term_node_tid_depth.inc
+files[] = views/nexteuropa_newsroom_views_handler_field_taxonomy.inc
+files[] = classes/NewsroomHelper.inc
```

Comment:
The rest of the custom classes are not mentioned here as items of the files array, shouldn't be? The pattern of AgendaFactory is a nice one, but i'm not fully sure it shouldn't be mentioned here for drupal core to map them all.